method edits for readability (#220)
greenelab/covid19-review: content/60.methods.md

The authors solicited relevant articles to be submitted via [GitHub](https://github.com/greenelab/covid19-review) for review.
Articles were classified as _diagnostic_, _therapeutic_, or _other_.
Following a framework often used for assessing medical literature, the review consisted of examining methods used in each relevant article, assignment (whether the study was observational or randomized), assessment, results, interpretation, and how well the study extrapolates [@doi:10.5014/ajot.60.4.367].

#### Diagnostic Papers

##### Methods

Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, countries / regions considered in case of human subjects, demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.
They then described the reference test or "gold standard," if one was utilized.

##### Assignment

Reviewers described how new and reference tests were assigned, including additional relevant details about the study design.

> For example, reviewers may evaluate whether the diagnostic test resulted in rigorous assignments of case status or was biased towards sicker or healthier individuals.

Reviewer: ```suggestion
For example, reviewers were asked whether the diagnostic test resulted in rigorous assignments of case status or was biased towards sicker or healthier individuals.
```
